lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -1060,7 +1060,8 @@ def get(self, request, subsection_id):
             original_grade = PersistentSubsectionGrade.read_grade(user_id, usage_key)
             if original_grade is not None and hasattr(original_grade, 'override'):
                 override = original_grade.override
-                history = list(original_grade.override.history.all().order_by('history_date')[:history_record_limit])
+                history = list(original_grade.override.history.filter(grade_id=original_grade.id).
```

Comment:
Just a stylistic change - please break lines after `(` instead of `.`, maybe like this:
```python
history = list(
    original_grade.override.history.filter(
        grade_id=original_grade.id
    ).order_by(
        'history_date'
    )
)[:history_record_limit]
```
I know it's more lines, but it's easier to read and adheres to our style guide: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines

Reply:
I had some issue with local testing, but it appears to work correctly. Will do style change.